queries/currency.py

```diff
@@ -205,10 +226,17 @@ def help_text(lemma):
 
 QCurCAD/fall →
     | 'kanadískur:lo'_kk/fall 'dollari:kk'/fall
+    | 'kanadadalur:no'_kk/fall
```

Comment:
These forms are not in BÍN (but should be!)  Also, it should be `'kanadadalur:kk'/fall`, not `:no`